widgetastic_manageiq.py

```diff
@@ -1309,3 +1309,46 @@ def all_emails(self):
 
     def read(self):
         return list(self.all_emails)
+
+
+class Slider(Widget):
+    """This control represents HTML5 Slider
+
+    """
+    ROOT = ParametrizedLocator('{@locator}')
+
+    def __init__(self, parent, locator, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+        self.locator = locator
+
+    @property
+    def value(self):
+        return float(self.browser.get_attribute('value', self))
+
+    @property
+    def max(self):
+        return float(self.browser.get_attribute('max', self))
+
+    @property
+    def min(self):
+        return float(self.browser.get_attribute('min', self))
+
+    @property
+    def step(self):
+        return float(self.browser.get_attribute('step', self))
+
+    @value.setter
+    def value(self, val):
+        self.browser.set_attribute('value', val, self)
```

Comment:
**REQUIRED**: This setter doesn't work. It will not change "value" field in GUI. It must be changed